### PR TITLE
R20.1 erroneous prompt on project load

### PR DESCRIPTION
**Pull request by bradymadams**

### CODE REVIEW

File: SmartSlicePlugin/utils/__init__.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def getPrintableNodes():
 def getModifierMeshes():
     return getNodes(
         lambda isSliceable, isPrinting, isSupport, isInfillMesh: \
-            isSliceable and not isPrinting and not isSupport and isInfillMesh
+            isSliceable and not isSupport and isInfillMesh
```
Comment (colman-hartinger) [inline SmartSlicePlugin/utils/__init__.py:82]: you can remove the `isPrinting` from the lambda

Comment (bradymadams) [reply SmartSlicePlugin/utils/__init__.py:82]: I don't think so. `getNodes` still sends it as an argument.

**Review by dougkenik — APPROVED** [review]
Needs testing on an optimization result with and without mod meshes

File: SmartSlicePlugin/utils/__init__.py (hunk 1 above)
Comment (bradymadams) [reply SmartSlicePlugin/utils/__init__.py:82]: @colman-hartinger see here for what I mean

https://github.com/tetonsim/is-cura-ui/blob/07363a4240fc53e36cfac99920e09e04a27e2ec0/SmartSlicePlugin/utils/__init__.py#L66

**Review by colman-hartinger — APPROVED** [review]
Tested with a couple different files and everything seems to be working as expected now

**Comment by jake-kirkland-teton** [comment]
I'm currently eating, but I trust you three with the testing.

**Review by jake-kirkland-teton — APPROVED** [review]

**Pull request merged**